src/components/bulkConcepts/addBulkConcepts.jsx

```diff
@@ -60,6 +62,7 @@ export class AddBulkConcepts extends Component {
       });
     }
   };
+
```

Comment:
Is this part of this ticket?

Reply:
It's formatting from the linter. it now removed